flink-table/flink-sql-parser/pom.xml

```diff
@@ -193,6 +204,10 @@ under the License.
 					<groupId>org.apiguardian</groupId>
```

Comment:
Is this exclusion still valid because it seems Calcite uses apiguardian in version 1.1.2 now?

Reply:
1.1.2 appeared with Calcite 1.31.0 at https://issues.apache.org/jira/browse/CALCITE-5196
since this PR is about update only to 1.27.0 then it is still valid

Reply:
But in `flink-table/flink-table-planner/pom.xml` you set it to 1.1.2 explicitly, maybe we should add some comment in this file for the reason.

Reply:
I've just made it available for tests here
In fact 1.1.2 is used because of junit-jupiter 5.8.2 since this commit https://github.com/apache/flink/pull/17728

Reply:
It would say it make sense. The more helpful comments in pom files, the better. They get messy quite quickly which is why we constantly need to try to keep them readable.

Reply:
added a comment in `flink-table/flink-table-planner/pom.xml`